application-admintools-ui/src/main/resources/AdminTools/Code/Translations.xml

```diff
@@ -36,31 +36,64 @@
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
   <content>##Admin section
+AdminTools.Code.ConfigurationClass_excludedLines=Excluded lines
+AdminTools.Code.ConfigurationClass_excludedLines.hint=Lines containing sensitive info in xwiki.cfg and xwiki.properties files
 AdminTools.Code.ConfigurationClass_serverLocation=Server location
 AdminTools.Code.ConfigurationClass_serverLocation.hint=Path to the server location
 admin.admintools.description=Configure Admin Tools Application
+
+## Modals
+adminTools.dashboard.download.modal.button=Download
+adminTools.dashboard.download.modal.date.from=From
+adminTools.dashboard.download.modal.date.to=To
+adminTools.dashboard.download.modal.provided=Configuration info
+adminTools.dashboard.download.modal.title=Get support files
+adminTools.dashboard.download.modal.xwikiConfig.title=xwiki.cfg
+adminTools.dashboard.download.modal.xwikiProperties.title=xwiki.properties
+adminTools.dashboard.download.modal.logs.title=Logs
+adminTools.dashboard.download.modal.logs.filter.placeholder.from=Logs from date...
+adminTools.dashboard.download.modal.logs.filter.placeholder.to=Logs to date...
+adminTools.dashboard.logs.modal.submit=View
+adminTools.dashboard.logs.modal.nLines.label=Number of extracted lines
+adminTools.dashboard.logs.modal.title=Show last n lines of log
+adminTools.dashboard.logs.modal.nLines.hint=Limited at 50000 lines. If the input is empty, the default of 1000 lines will be returned.
```

Comment:
`The maximum number of lines accepted is 50000, since the complete logs can also be downloaded. If no value is defined, 1000 will be used as a default value.`